QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/layout.js

```
@@ -0,0 +1,64 @@
+        }
+
+        // Start blink animation
+        $balanceContainer.addClass("blink");
+
+        $.ajax({
+            url: '/User/Wallet/GetBalance',
+            type: 'GET',
+            dataType: 'json',
+            success: function (response) {
+                $balanceContainer.removeClass("blink");
+                const finalBalance = parseFloat(response.balance);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)